packages/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx

```diff
@@ -536,11 +536,16 @@ export class VerticalBarChartBase extends React.Component<IVerticalBarChartProps
         shouldHighlight: shouldHighlight,
       });
       const barHeight: number = Math.max(yBarScale(point.y), 0);
-      if (barHeight < 1) {
+      let adjustedBarHeight = 0;
+      if (barHeight <= 0) {
         return <React.Fragment key={point.x}> </React.Fragment>;
+      } else if (barHeight <= Math.ceil(yBarScale(this._yMax) / 100.0)) {
+        adjustedBarHeight = Math.ceil(yBarScale(this._yMax) / 100.0);
+      } else {
```

Comment:
Are you willing to fix the same issue in vertical stacked bar chart as well.

Reply:
Yes... Sure... 
Will that be covered as a separate issue or will it be covered in this PR itself??

Reply:
You can make the changes in the same PR.

Reply:
Cool... Making the changes then

Reply:
<img width="313" alt="image" src="https://github.com/microsoft/fluentui/assets/35366351/2b6cd3c5-ad02-4788-8ec0-1f10985effb5">
I have found an existing code segment which hints at a possible implementation for the feature for stacked bar charts...
Does the issue actually exist for stacked case?
Could you please have a look?

Reply:
Seems that it is passed as a prop??

<img width="433" alt="image" src="https://github.com/microsoft/fluentui/assets/35366351/8d06a30d-5b25-4ede-9967-5e550c629b55">

Reply:
So we wish to have a min Height for 0 value for the stacked case specially... for visual depiction of an data class existing but zero... 
Should barMinimumHeight(which is passed as prop) have the 1% height of _yMax as its lowest bound as well...

Reply:
Yes, same behavior as vertical bar chart. Any height from 0 to 1% to have a fixed height of 1%.
Right now barHeight and barMinimumHeight are all defined in pixels not in percentages.

Reply:
Cool... So even if by passed props, barMinimumHeight is zero, we should enforce the 1% min height rule for the stacked chart?